client/components/mma/shared/SupporterPlusTsAndCs.tsx

```diff
@@ -37,8 +37,8 @@ export const SupporterPlusTsAndCs = ({
 	const currencySymbol = convertCurrencyToSymbol(currencyISO);
 	const monthlyOrAnnual =
 		calculateMonthlyOrAnnualFromBillingPeriod(billingPeriod);
-	const monthlyThreshold = getBenefitsThreshold(currencyISO, 'Monthly');
```

Comment:
Just curious why this renaming?

Reply:
the equivalent config for contributions amounts used month / year instead of monthly / annual, so I made it more consistent but it also avoids having to calculate `monthlyOrAnnual` to get the config